### DISCUSSION TITLE
How to upgrade taqueria?

### DISCUSSION THREAD

**Discussion by crcdng** [ecadlabs/taqueria#967] [Ideas]
Hi, 

If I didn't miss anything, currently upgrading a project to the latest version of taqueria requires a manual reinstall. Then  plugins would have to be uninstalled, followed by installations (?)

May I suggest an upgrade feature, that upgrades taqueria including plugins in the current project?

**Comment by russellmorton** [comment]
Upgrading Taqueria is definitely an important part of the developer experience for Taqueria. Having a user-friendly way to do this is a great idea.

I have added this to our backlog (#198).

Thank you @i3games!
